### PR TITLE
Add logging and comments (from review of #202)

### PR DESCRIPTION
**Pull request by rpanak-generalbytes**

### CODE REVIEW

File: src/main/java/bitbucketpullrequestbuilder/bitbucketpullrequestbuilder/BitbucketRepository.java [hunk 1]
```diff
@@ -158,16 +158,25 @@ private BitbucketCause createCause(AbstractPullrequest pullRequest) {
         String repoUri = null;
 
         if(links != null) {
-            final List<AbstractPullrequest.RepositoryLink> cloneLinks = links.getClone();
+            final List<AbstractPullrequest.RepositoryLink> cloneLinks = links.getCloneLinks();
```
Comment (CodeMonk) [inline src/main/java/bitbucketpullrequestbuilder/bitbucketpullrequestbuilder/BitbucketRepository.java]: Did you miss this one in your  original PR?

Comment (rpanak-generalbytes) [reply src/main/java/bitbucketpullrequestbuilder/bitbucketpullrequestbuilder/BitbucketRepository.java]: In the original PR you commented that the original field name is confusing (https://github.com/nishio-dens/bitbucket-pullrequest-builder-plugin/pull/202#discussion_r304996964), this is just the result of the rename.

Comment (CodeMonk) [reply src/main/java/bitbucketpullrequestbuilder/bitbucketpullrequestbuilder/BitbucketRepository.java]: 👍

**Comment by CodeMonk** [comment]
I have just merged a fix for the build into master. Please rebase or pull those changes into your PR to force a new build (that will, hopefully, pass)

**Comment by rpanak-generalbytes** [comment]
done

**Pull request merged**